### PR TITLE
SNOW-1518540 release public preview

### PR DESCRIPTION
**Pull request by sfc-gh-zli**
# Overview

SNOW-1518540 release public preview

## Pre-review checklist

- [ ] I attest that this change meets the bar for low risk without security requirements as defined in the [Accelerated Risk Assessment Criteria](https://developer-handbook.m1.us-west-2.aws.app.snowflake.com/docs/reference/security-review/accelerated-risk-assessment/#eligibility) and I have taken the [Risk Assessment Training in Workday](https://wd5.myworkday.com/snowflake/learning/course/6c613806284a1001f111fedf3e4e0000).
    - Checking this checkbox is mandatory if using the [Accelerated Risk Assessment](https://developer-handbook.m1.us-west-2.aws.app.snowflake.com/docs/reference/security-review/accelerated-risk-assessment/) to risk assess the changes in this Pull Request.
    - If this change does not meet the bar for low risk without security requirements (as confirmed by the peer reviewers of this pull request) then a [formal Risk Assessment](https://developer-handbook.m1.us-west-2.aws.app.snowflake.com/docs/reference/security-review/risk-assessment/) must be completed. Please note that a formal Risk Assessment will require you to spend extra time performing a security review for this change. Please account for this extra time earlier rather than later to avoid unnecessary delays in the release process.
- [ ] This change has passed precommit
- [ ] This change is TEST-ONLY
- [ ] This change has been through a design review process and the test plan is signed off by the owner
- [ ] This change is parameter protected
- [ ] This change has code coverage for the new code added

### CODE REVIEW

**Review by sfc-gh-vpai — CHANGES_REQUESTED** [review]
I think the top-level `README.md` probably needs to change as well since it refers to the `private-preview` directory

File: public-preview/8.30/specification/database.yaml [hunk 1]
```diff
@@ -0,0 +1,793 @@
...
+        in: query
+        description: Optionally includes dropped databases that have not yet been
+          purged.
+        schema:
+          type: boolean
+      responses:
+        '200':
+          description: Successful request.
+          headers:
+            X-Snowflake-Request-ID:
```
Comment (sfc-gh-igarish) [inline public-preview/8.30/specification/database.yaml:37]: Are we keeping X- header here?

**Review by sfc-gh-vpai — APPROVED** [review]

**Review by sfc-gh-wshangguan — COMMENTED** [review]
For some specs, e.g. dynamic table, is it the manual created version or the one generated by DSL?

**Comment by sfc-gh-zli** [comment]
> For some specs, e.g. dynamic table, is it the manual created version or the one generated by DSL?

It's the one in 8.30 release branch. I think it is generated, because it has x-enum-varname.

**Pull request merged**